ami/models/components/unet/commons.py

```diff
@@ -0,0 +1,146 @@
+# Ref: https://github.com/facebookresearch/RCDM
+
+import math
+
+import torch
+import torch.nn as nn
+
+
+class ResBlock(nn.Module):
+    """A residual block for Unet Components."""
+
+    def __init__(
+        self,
+        in_channels: int,
+        emb_channels: int,
+        out_channels: int,
+    ) -> None:
+        """A residual block for Unet Components.
+
+        Args:
+            in_channels (int):
+                Input num of channels.
+            emb_channels (int):
+                Input embedding num of channels.
+            out_channels (int):
+                Output num of channels.
+        """
+        super().__init__()
+        self.in_layers = nn.Sequential(
+            nn.GroupNorm(num_groups=32, num_channels=in_channels),
```

Comment:
GroupNormの`num_groups`で`num_channels`は必ず割り切れないといけないから、コンストラクタの引数に`num_norm_groups=32`を追加してそれを指定する形にしよう。
```suggestion
            nn.GroupNorm(num_groups=num_norm_groups, num_channels=in_channels),
```